src/buffalo/buffalo.ts

```diff
@@ -91,6 +91,30 @@ class Buffalo {
         return value;
     }
 
+    public writeUInt56(value: bigint): void {
+        this.buffer.writeUIntLE(Number(value & 0xffffffffffffn), this.position, 6);
+        this.buffer.writeUint8(Number(value >> 48n), this.position + 6);
```

Comment:
Being picky: should use `writeUIntX` format for consistency across the file (couple of occurrences use `writeUintX`). 😛